watson-developer-cloud/discovery-components
# fix: search facet text not wrapping

#### What do these changes do/fix?
Allows the search facet text to wrap onto the next line if the string is too long. Some styling changes are also added as a PR in the tooling repo for the radio buttons.
**Before:**
<img width="412" alt="Screen Shot 2020-02-28 at 2 50 28 PM" src="https://user-images.githubusercontent.com/59846843/75707357-5d5c7400-5c84-11ea-8421-c1c9ecdf430b.png">
**After:**
<img width="409" alt="Screen Shot 2020-03-02 at 12 10 24 PM" src="https://user-images.githubusercontent.com/59846843/75707301-4ae23a80-5c84-11ea-9ba3-f17f0ea25011.png">

#### How do you test/verify these changes?

- Link this PR with the corresponding one in tooling.
- In the Improve and Custom section of a project add Customize Display facets with longer strings.
- See that the overflow text wraps.

#### Are there any breaking changes included in this pull request?
No

Connecting issue: https://github.ibm.com/Watson-Discovery/the-tooling-core-tracker/issues/1201
